### DISCUSSION TITLE
Help getting started on Windows with Budibase CLI

### DISCUSSION THREAD

**Discussion by Chris9soul** [Budibase/budibase#1635] [Help] [answered]
So, now with the new release, I tried to use the new platform, but found the installation process kinda complicated. I downloaded docker but could not figure out how to run the commands.

I'm not that great with code so any help would be greatly appreciated.

**Comment by shogunpurple** [comment, marked as answer]
@Chris9soul 

You need to run your commands in the terminal application on your machine. Open the terminal and follow the guide at https://docs.budibase.com/getting-started.

If you are not comfortable with the command line, you would likely be better served by our one click digitalocean offering. I will caveat this by saying that we are waiting for digitalocean to approve our latest release, which should happen over the next few days.

https://docs.budibase.com/self-hosting/hosting-methods/digitalocean

**Reply by Chris9soul** [reply]
So do I just open the Command Prompt and write that code?
If so, I had trouble running the CLI installment. Is this not what I need to run?:

wget https://github.com/Budibase/budibase/releases/download/v0.9.1/cli-win.exe

chmod +x cli-win.exe


mv cli-win.exe budi

**Reply by shogunpurple** [reply]
@Chris9soul 

- Download https://github.com/Budibase/budibase/releases/download/v0.9.1/cli-win.exe into a folder on your machine
- open powershell
- `cd` into the directory that you installed the cli file above. For example, if it's in `Downloads`, `cd .\Downloads\`
- `.\cli.win.exe hosting --init`
- `.\cli.win.exe hosting --start`

Budibase should now be running on localhost:10000.

After having done this setup, you should be able to start/stop the budibase containers from docker desktop in future.

**Reply by Chris9soul** [reply]
Thank you! Much clearer!

**Reply by sanjeethflautist** [reply]
Hi I tried same and i am getting below error:
PS C:\development\budibase> C:\development\budibase\cli-win.exe hosting --init
? This will create multiple files in current directory, should continue? Yes
Error - Error: self signed certificate in certificate chain
PS C:\development\budibase>
